bidi-scanner/github/dist/licenses.txt

```diff
@@ -0,0 +1,302 @@
+@actions/core
```

Comment:
Note that this file and the two above are in the `dist` directory. The `index.mjs` file contains not only the `github` directory code, but also the code from all the dependencies. They were bundled together with webpack, which in turn was invoked using Vercel's `ncc` tool (see the `build` script in `package.json`).